fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -567,75 +567,77 @@ private ResourceDAO getResourceDao() {
         try {
             existingResourceDTO = this.getResourceDao().read(logicalId, resourceType.getSimpleName());
 
-            if (existingResourceDTO != null) {
-                if (existingResourceDTO.isDeleted()) {
-                    existingResource = this.convertResourceDTO(existingResourceDTO, resourceType, null);
-                    resourceBuilder = existingResource.toBuilder();
-
-                    SingleResourceResult<T> result = new SingleResourceResult.Builder<T>()
-                            .success(true)
-                            .resource(existingResource)
-                            .build();
+            if (existingResourceDTO == null) {
+                throw new FHIRPersistenceResourceNotFoundException("resource does not exist: " + 
+                        resourceType.getSimpleName() + ":" + logicalId);
```

Comment:
2 ":" seems a little bit strange, can we change to 
resourceType.getSimpleName() + ":" + logicalId" + "does not exist." 

or use the same style as the other warning info of the following codes.

Reply:
Adding to Albert's comments... we should bracket the values in the message.

Reply:
My PR didn't actually change the error message from before, but I agree with these comments and will address now.

Reply:
I think I'll just change it to `"resource does not exist: " + resourceType.getSimpleName() + "/" + logicalId)`
I usually like to bracket the values in `'` but in this case the values can't contain spaces